mininapse.css

```diff
@@ -393,3 +393,8 @@ code {
 #upload-progressbar {
     background: var(--highlight-border-color);
 }
+
+/* Scrollbars */
+body {
+    scrollbar-color: var(--highlight-border-color) var(--body-bg-color);
```

Comment:
```suggestion
    scrollbar-color: rgba(255, 255, 255, 0.06) var(--body-bg-color);
```

Reply:
It should have at least a 3:1 contrast ratio to comply with WCAG

Reply:
mmmh. I am not a pro on accessibility but doesn't that guideline only apply to text? 

as a user with no visual impairment that red is way too distracting for me, not sure how others (@fnutt ?) feel. I personally don't use firefox so I don't mind whatever color we choose, chrome already has very dim scroll bars.

Reply:
There's a sidenote on the CSS Scrollbars Module document

> When using scrollbar-color property with specific color values, authors should ensure the specified colors have enough contrast between them. For keyword values, UAs should ensure the colors they use have enough contrast. See Techniques for WCAG 2.0: G183: Using a contrast ratio of 3:1 […] [WCAG20].
https://www.w3.org/TR/2018/WD-css-scrollbars-1-20180925/#scrollbar-color

Reply:
Sorry for late response.

I don't like the red colour that much either. It's very "in your face" for something that I would not like to be in my face.